gamedata/LOUD_Misc/mods/SupremeScoreBoard/modules/ext.strings.lua

```diff
@@ -156,29 +158,37 @@ function initalize()
     locals['ur_SALVAMAVOSCATH_info'] = 'AEON Salvation \n CYBRAN Scathis '
     locals['ur_ENGISTATION'] = 'No Engineering Station'
     locals['ur_PRODFA'] = 'No FA Units'
-    locals['ur_T1']    = 'No TECH 1 Units'
-    locals['ur_T2']    = 'No TECH 2 Units'
-    locals['ur_T3']    = 'No TECH 3 Units'
+    locals['ur_T1']    = 'No Tech 1'
+    locals['ur_T2']    = 'No Tech 2'
+    locals['ur_T3']    = 'No Tech 3'
     locals['ur_T3AIR'] = 'No TECH 3 Air Units'
-    locals['ur_EXPERIMENTAL'] = 'No TECH 4 Units (Exp)'
-    locals['ur_T3MOBILEAA'] = 'No T3 Mobile Air Units'
+    locals['ur_EXPERIMENTAL'] = 'No Experimental/Tech 4'
     locals['ur_WALL'] = 'No Wall Structures'
     locals['ur_SUPPFAC'] = 'No Support Factories'
-    locals['ur_NAVAL'] = 'No Naval Units'
+    locals['ur_NAVAL'] = 'No Naval'
     locals['ur_AIR']   = 'No Air Units'
-    locals['ur_LAND']  = 'No Land Units'
-    locals['ur_SUPCOM'] = 'No Support Commanders'
+    locals['ur_LAND']  = 'No Land'
+    locals['ur_SUPPORTCOMMANDERS'] = 'No Support Commanders (SACUs)'
     locals['ur_EYE'] = 'No Super Intel'
     locals['ur_EYE_info'] = 'AEON Rhianne \n CYBRAN Soothsayer'
     locals['ur_INTEL'] = 'No Intel Structures'
-    locals['ur_INTEL_info'] = 'No Sonars, Radars, Optical, Sensors'

```

Comment:
"Optical" is too vague here to make sense to players who don't already know about the lineup of special intel structures (Eye of Rhianne, Soothsayer, etc.) but this description should point out that these buildings are disabled. Maybe universally use the wording "No Intel Structures"? Feel free to suggest alternatives.

Reply:
The title of the restriction which the player will see in-game is "No Intel Structures" (new line 174). Anything with "_info" is the description of what it's referring to, like how the descriptions in tooltips.lua ties back to the titles in LEMLobbyOptions.lua.

That being said, I don't see when these "_info" descriptions are shown in-game, so all of them can probably be removed.

Reply:
I wasn't paying enough attention to realise this earlier, but these strings are used by the indicators Supreme Score Board displays about the rules of the currently-ongoing game, although the LOUD version I've become familiar with generally doesn't do any such thing.

I'm not strictly sure why not, but it wouldn't be a bad idea to put them back in if it's practical. I may investigate to see whether they were removed for a specific reason.

In any case, if you're already covering them, just make sure they're consistent with all the other variations before we merge.